extensions/quickgpt/src/index.tsx

```diff
@@ -0,0 +1,293 @@
+import { useState } from "react";
+import {
+  List,
+  ActionPanel,
+  Action as RaycastAction,
+  Clipboard,
+  LaunchProps,
+  Icon,
+  Action,
+  Color,
+  closeMainWindow,
+  getPreferenceValues,
+} from "@raycast/api";
+import { runAppleScript } from "@raycast/utils";
+import pinsManager from "./pinsManager";
+import promptManager, { PromptProps } from "./promptManager";
+import { contentFormat } from "./contentFormat";
+import fs from "fs";
+import path from "path";
+import { match } from "pinyin-pro";
+import useClipboardAndSelectionText from "./useClipboardAndSelectionText";
+import React from "react";
+
+const IDENTIFIER_PREFIX = "quickgpt-";
+const DEFAULT_ICON = "🔖";
+
+const supportedPrefixCMD: { [key: string]: string } = {
+  c: "中文作答",
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English.